brave/gulpfile.js/brave-locales.js

```diff
@@ -0,0 +1,23 @@
+const gulp = require('gulp')
+const jsoneditor = require('gulp-json-editor')
+
+const createBraveLocalesTask = () =>
+  gulp.task('locales:brave', function () {
+    return gulp.src('./dist/brave/_locales/**/*')
+    .pipe(jsoneditor(function (json) {
+      Object.keys(json).forEach((stringName) => {
+        if (json[stringName]) {
```

Comment:
Just curious, what makes this check necessary? Are some props set to `undefined` or `false`?

Reply:
it might not be necessary, I was probably just being overly robust, I'll remove it.

Reply:
fixed